openhantek/src/hantekdso/hantekdsocontrol.cpp

```diff
@@ -1223,8 +1233,8 @@ void HantekDsoControl::run() {
             this->_samplingStarted = false;
 
             // Start next capture if necessary by leaving out the break statement
-
             if (!this->sampling) break;
+#if defined(__clang__)
```

Comment:
Which compiler do you use? This part works with G++ and clang and if another special case is required, we should add it. C++14 added the fallthrough attribute officially.

Reply:
Gcc 4.8.5 and failed to interpret the subsequent __has... so i had to add a check to compile

Reply:
Understandable. G++ 4.8.5 is only 2 1/2 years old, that can and should still be supported. Leave it as it is and I will find a solution in a later PR.